docs/site/tutorials/todo/todo-tutorial-geocoding-service.md

```diff
@@ -93,13 +94,13 @@ docs here: [REST connector](/doc/en/lb3/REST-connector.html).
 
 ### Implement a service provider
 
-Use the `lb4 service` command and the following inputs to create a calculator
+Use the `lb4 service` command and the following inputs to create a geocoder
 service:
 
 ```sh
 lb4 service
 ? Service type: Remote service proxy backed by a data source
-? Please select the datasource GeocoderDataSource
+? Please select the datasource GeocoderDatasource
```

Comment:
When I create the `geocoder` service, it creates
```
Service Geocoder was created in src/services/
```

and the content looks like this:

```
export interface Geocoder {
  // this is where you define the Node.js methods that will be
  // mapped to REST/SOAP/gRPC operations as stated in the datasource
  // json file.
}

export class GeocoderProvider implements Provider<Geocoder> {
  constructor(
    // geocoder must match the name property in the datasource json file
    @inject('datasources.geocoder')
    protected dataSource: GeocoderDataSource = new GeocoderDataSource(),
  ) {}

  value(): Promise<Geocoder> {
    return getService(this.dataSource);
  }
}

```

Notice `Geocoder` is the interface.  But the code shown in the snippets show the interface as 
`GeocoderService`.

Is this a regression in the CLI (to no longer append 'Service' at the end of the name the user enters), or is this just an old documentation typo?

@raymondfeng , @bajtos ?

Reply:
@nora does the geo service work?
I am getting an empty array returned for a post of 
```
{
  "title": "go to disney world",
  "desc": "Magic Kingdom",
  "isComplete": false,
  "remindAtAddress": "1180 Seven Seas Drive,  Lake Buena Vista, FL  32830",
  "remindAtGeo": "string"
}
```

Reply:
> Is this a regression in the CLI (to no longer append 'Service' at the end of the name the user enters), or is this just an old documentation typo?

Before it wasn't created with the CLI so that's probably why, but I'll update the example and docs to use `GeocoderProvider` as the class name.

Reply:
> @nabdelgadir 
 does the geo service work?
> I am getting an empty array returned for a post of
> 
> ```{
>   "title": "go to disney world",
>   "desc": "Magic Kingdom",
>   "isComplete": false,
>   "remindAtAddress": "1180 Seven Seas Drive,  Lake Buena Vista, FL  32830",
>   "remindAtGeo": "string"
> }```

It does work but sometimes it can't find the address. If you try with `1 New Orchard Road, Armonk, 10504` instead it should work. There's this comment in the Todo controller for this: `// TODO handle "address not found"`

Reply:
For the prompt for `lb4 relation` for hasMany, I see in docs

> ? Foreign key name to define on the source model todoListId

but real prompt says:

? Source property name for the relation getter (todoListId)  

The prompt doesn't show:

```
? Relation name todoList
```

but I see it in the docs.

In total, for belongsTo, I see these prompts:

```
dominiques-mbp:todo-list dremond$ lb4 relation
? Please select the relation type belongsTo
? Please select source model Todo
? Please select target model TodoList
? Source property name for the relation getter todoListId
? Allow Todo queries to include data from related TodoList instances? Yes
   create src/controllers/todo-todo-list.controller.ts

Relation BelongsTo was created in src/
```

Reply:
> For the prompt for `lb4 relation` for hasMany, I see in docs
> 
> > ? Foreign key name to define on the source model todoListId
> 
> but real prompt says:
> 
> ? Source property name for the relation getter (todoListId)
> 
> The prompt doesn't show:
> 
> ```
> ? Relation name todoList
> ```
> 
> but I see it in the docs.
> 
> In total, for belongsTo, I see these prompts:
> 
> ```
> dominiques-mbp:todo-list dremond$ lb4 relation
> ? Please select the relation type belongsTo
> ? Please select source model Todo
> ? Please select target model TodoList
> ? Source property name for the relation getter todoListId
> ? Allow Todo queries to include data from related TodoList instances? Yes
>    create src/controllers/todo-todo-list.controller.ts
> 
> Relation BelongsTo was created in src/
> ```

They were updated in https://github.com/strongloop/loopback-next/pull/4373

Reply:
@emonddr Please use `npm link` to use the latest code.